src/tree_sortable.rs

```diff
@@ -0,0 +1,106 @@
+// This file was generated by gir (17af302) from gir-files (11e0e6d)
+// DO NOT EDIT
+
+use SortType;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use auto::traits::TreeSortableExt as Auto;
+use std::thread;
+use std::cell::RefCell;
+use std::mem::transmute;
+use std::process;
+
+use glib_ffi::gpointer;
+use {TreeSortable, TreeIter};
+use ffi::{GtkTreeIter, GtkTreeSortable};
+
+pub trait TreeSortableExt {
+    // -- manual --
+    fn set_default_sort_func<F>(&self, sort_func: F)
+    where F: FnMut(&TreeSortable, &TreeIter, &TreeIter) -> i32 + 'static;
```

Comment:
This is about the first parameter. If I have `store: TreeStore` then `store.set_default_sort_func()` will take `FnMut(&TreeStore, &TreeIter, &TreeIter) -> Ordering`. If I have `store: TreeSortable`, it will be `FnMut(&TreeSortable, &TreeIter, &TreeIter) -> Ordering`.